### PR TITLE
[disasters] small fixes

### PR DESCRIPTION
**Pull request by beets**
- add contained_place_types overrides for more countries
- filter child places which are unsupported
- remove logging statements from #2461 (the issue didn't reproduce anymore, leading me to think that we were originally sampling too many places and timing out occasionally)
- update place type for the page from list to str

### CODE REVIEW

File: server/routes/event.py [hunk 1]
```diff
@@ -192,29 +192,19 @@ def event_node(dcid=DEFAULT_EVENT_DCID):
...
+        place_metadata.contained_place_types)
+    contained_place_type = place_metadata.contained_place_types.get(
+        place_metadata.place_type, None)
     subject_config = lib_subject_page_config.remove_empty_charts(
         subject_config, place_dcid, contained_place_type)
 
     # TODO: If not enough charts from the current place, add from the next place up and so on.
     subject_page_args = {
-        "place_type": json.dumps(place_metadata.place_types),
+        "place_type": json.dumps(place_metadata.place_type),
```
Comment (chejennifer) [inline server/routes/event.py:207]: nit: looks like place_type is just a string now right? so don't need to json.dumps

File: server/lib/subject_page_config.py [hunk 2]
```diff
@@ -236,27 +247,45 @@ def place_metadata(place_dcid) -> PlaceMetadata:
...
+
+  filtered_child_places = {}
+  if get_child_places:
+    child_places = place_api.child_fetch(place_dcid)
+    for place_type in child_places:
+      child_places[place_type].sort(key=lambda x: x['pop'], reverse=True)
+      child_places[place_type] = child_places[place_type][:place_api.
+                                                          CHILD_PLACE_LIMIT]
+
+    # Filter child places for unsupported place types
```
Comment (chejennifer) [inline server/lib/subject_page_config.py:272]: nit: ...for supported place types?

**Review by chejennifer — APPROVED** [review]
nice, thanks for the fix!

File: server/routes/event.py (hunk 1 above)
Comment (beets) [reply server/routes/event.py]: thanks for catching that

File: server/lib/subject_page_config.py (hunk 2 above)
Comment (beets) [reply server/lib/subject_page_config.py]: done

**Comment by beets** [comment]
thanks for the review!

**Pull request merged**